util/issue_commands.go

```diff
@@ -43,17 +47,28 @@ func SendRDDLReissuanceResult(ctx sdk.Context, proposerAddress string, txID stri
 	cmd := exec.Command("planetmint-god", "tx", "dao", "reissue-rddl-result",
 		"--from", sendingValidatorAddress, "-y",
 		proposerAddress, txID, strconv.FormatInt(blockHeight, 10))
+	logger.Debug("REISSUE: create Result")
+	return execCommand(cmd)
+}
 
-	var stdout, stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	// Start the command in a non-blocking way
-	err := cmd.Start()
-	errstr := stderr.String()
-	if err != nil || len(errstr) > 0 {
-		if err == nil {
-			err = errors.New(errstr)
-		}
-	}
-	return err
+func SendRDDLDistributionRequest(ctx sdk.Context, distribution types.DistributionOrder) error {
+	logger := ctx.Logger()
+	// Construct the command
+	sendingValidatorAddress := config.GetConfig().ValidatorAddress
+	cmd := exec.Command("planetmint-god", "tx", "dao", "distribution-request",
+		"--from", sendingValidatorAddress, "-y", "'"+distribution.String()+"'")
+	logger.Debug("REISSUE: create Result")
+	return execCommand(cmd)
+}
+
+func SendRDDLDistributionResult(ctx sdk.Context, lastPoP string, daoTxid string, invTxid string, popTxid string) error {
+	logger := ctx.Logger()
+	// Construct the command
+	sendingValidatorAddress := config.GetConfig().ValidatorAddress
+	logger.Debug("REISSUE: create Result")
+	cmd := exec.Command("planetmint-god", "tx", "dao", "distribution-result",
```

Comment:
normally I would reject that, but since i am currently moving planetmint-god tx... calls in this file to (g)RPC I leave this one alone.